src/tests/ForecastSummary.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
This test suite is also failing for many of the same reasons outlined in `ForecastSummaries.test.js`.